drivers/scheduler/rke/rke.go

```diff
@@ -270,6 +270,53 @@ func (r *Rancher) SaveSchedulerLogsToFile(n node.Node, location string) error {
 	return err
 }
 
+// MoveNamespaceToNoProject moves namespace to no project
+func (r *Rancher) MoveNamespaceToNoProject(nsList []string) error {
+	for _, ns := range nsList {
+		ns, err := core.Instance().GetNamespace(ns)
+		if err != nil {
+			return err
+		}
+		delete(ns.Labels, "field.cattle.io/projectId")
+		delete(ns.Annotations, "field.cattle.io/projectId")
+		_, err = core.Instance().UpdateNamespace(ns)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// MoveNamespaceFromOneProjectToAnotherProject moves namespace from one project to other project
+func (r *Rancher) MoveNamespaceFromOneProjectToAnotherProject(projectName string, nsList []string) error {
```

Comment:
Can we call this `ChangeProjectForNamespace`?

Reply:
done